pyproject.toml

```diff
@@ -18,15 +18,18 @@ classifiers = [
   "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
   "Framework :: Django",
   "Framework :: Django :: 3.2",
   "Framework :: Django :: 4.0",
+  "Framework :: Django :: 4.1",
+  "Framework :: Django :: 4.2",
 ]
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
-Django = ">=3.0"
-strawberry-graphql = ">=0.139.0"
+Django = ">=3.2,<4.2"
+strawberry-graphql = "^0.154.0"
```

Comment:
This should be rolled back. As of `0.9.1`, `strawberry-graphql-django` depends on strawberry `>= 0.154, < 0.155`.

Reply:
You are right! I just published a new version removing the upper bound limit from both django and strawberry